data/json/items/book/misc.json

```
@@ -128,6 +128,22 @@
     "intelligence": 3,
     "time": 1
   },
+  {
+    "id": "mycenacean_hymns",
+    "type": "BOOK",
+    "name": "Mycenacean Hymns",
+    "description": "A vellum book containing the hymns central to Marloss faith.  As the verses lead to each other, the text sings of unity and promised paradise.",
```

```suggestion
    "description": "A vellum book containing the hymns central to Marloss faith. As the verses lead to each other, the text sings of unity and promised paradise.",
```
2 spaces after full stop.

---

Two spaces after full stop is standard, at least for item descriptions.
See [this review](https://github.com/CleverRaven/Cataclysm-DDA/pull/29776#discussion_r277155218).

---

Ha damn, sorry :( Thanks for the heads up.

---

Dont worry, its not an intuitive rule.